src/super_gradients/common/factories/transforms_factory.py

```diff
@@ -31,8 +37,12 @@ def __init__(self):
             "DetectionHorizontalFlip": DetectionHorizontalFlip,
             "DetectionPaddedRescale": DetectionPaddedRescale,
             "DetectionTargetsFormat": DetectionTargetsFormat,
-            "DetectionTargetsFormatTransform": DetectionTargetsFormatTransform
+            "DetectionTargetsFormatTransform": DetectionTargetsFormatTransform,
 
+            'RandomResizedCropAndInterpolation': RandomResizedCropAndInterpolation,
+            'color_augmentation': datasets_utils.get_color_augmentation,
```

Comment:
This is just a name we used previously (and a pretty bad one tbh) to get either RandAugment or color jitter.
We should get rid of this and set rand augment (also addd it to the factory) when we use it and color jitter separately.

Reply:
Changed accordingly